app/src/main/java/app/revanced/manager/ui/viewmodel/InstallerViewModel.kt

```diff
@@ -56,13 +57,36 @@ import java.util.logging.LogRecord
 class InstallerViewModel(
     private val input: Destination.Installer
 ) : ViewModel(), KoinComponent {
+    private val logger = ManagerLogger()
     private val app: Application by inject()
     private val fs: Filesystem by inject()
     private val pm: PM by inject()
     private val workerRepository: WorkerRepository by inject()
     private val installedAppRepository: InstalledAppRepository by inject()
     private val rootInstaller: RootInstaller by inject()
 
+    var packageInstallerResult by mutableStateOf<PackageInstallerResult?>(null)
+        private set
+
+    var showInstallerStatusDialog by mutableStateOf(false)
```

Comment:
I think we could just not show the dialog if `packageInstallerResult` is non-null instead of using an extra boolean.

Reply:
So whenever a result is present, a dialog is shown, which means we would need to delete the result to hide the dialog.

Reply:
Do you need to hide and then unhide a dialog? If not, it would work.